astroquery/atomic/core.py

```diff
@@ -39,7 +41,9 @@ def query_object(self, wavelength_range=None, wavelength_type=None,
                      show_auto_ionizing_transitions=None,
                      output_columns=('spec', 'type', 'conf',
                                      'term', 'angm', 'prob',
-                                     'ener')):
+                                     'ener'),
```

Comment:
we go up until line length of 120, please squash these into less lines to help readability.

Reply:
OK, but I consider this smuggling! =)